context_chat_backend/network_em.py

```diff
@@ -36,6 +39,9 @@ class NetworkEmbeddings(Embeddings, BaseModel):
 	def _get_embedding(self, input_: str | list[str]) -> list[float] | list[list[float]]:
 		emconf = self.app_config.embedding
 
+		lengths = [len(text) for text in (input_ if isinstance(input_, list) else [input_])]
+		logger.info(f'Sending embedding request for {len(lengths)} chunks of the following sizes: ', lengths)
```

Comment:
let's add the sum there too for a little convenience
`sizes and sum: ', lengths, sum(lengths))`

Reply:
:+1: